### PR TITLE
Added additional funcs to made possible run air with Rust

### PR DESCRIPTION
**Pull request by silvestrpredko**

### CODE REVIEW

File: crates/local-vm/src/local_vm.rs [hunk 1]
```diff
@@ -96,6 +96,9 @@ pub fn make_call_service_closure(
                 service_out.lock().extend(args.function_args);
                 ivalue_utils::unit()
             }
+            ("errorHandlingSrv", "error") => {
+                ivalue_utils::unit()
```
Comment (folex) [inline crates/local-vm/src/local_vm.rs:100]: I think that we need to at least log error here; At best - fail `receive_args` with that error.

Comment (silvestrpredko) [reply crates/local-vm/src/local_vm.rs:101]: @folex I have added a simple log, is it enough?

Comment (folex) [reply crates/local-vm/src/local_vm.rs:101]: Nope, you need to log the passed arguments. Like this:
```
log::warn!("caught an error: {:?}", args.function_args);
```

**Review by folex — COMMENTED** [review]
What about variable loading? 

If you just pass `hashmap!` with data, local_vm would insert `load` instructions before `getDataSrv` instructions and the script will be invalid.

You need something like this:
https://github.com/fluencelabs/fluence/pull/1110/files#diff-f0a2f5fe61a4b414b315b9d6011ff0a5e79423a03e3b94195cc7c3fae0039c2fR142

**Comment by silvestrpredko** [comment]
@folex You mean

> What about variable loading?
> 
> If you just pass `hashmap!` with data, local_vm would insert `load` instructions before `getDataSrv` instructions and the script will be invalid.
> 
> You need something like this:
> https://github.com/fluencelabs/fluence/pull/1110/files#diff-f0a2f5fe61a4b414b315b9d6011ff0a5e79423a03e3b94195cc7c3fae0039c2fR142

You mean I need to load variables inside make_call_service_closure?

**Comment by folex** [comment]
Superseded by #1143

**Pull request closed without merging**